docs/sphinx/source/whatsnew/v0.6.2.rst

```diff
@@ -14,7 +14,12 @@ date will require Python 3. (:issue:`501`)
 
 API Changes
 ~~~~~~~~~~~
-
+* `erbs` `doy` argument changed to `datetime_or_doy` to be consistent with
+  allowed types and similar functions (`disc`, `get_extra_radiation`).
+* `erbs` DataFrame vs. OrderedDict return behavior now determined by type
+  of `datetime_or_doy` instead of `ghi` or `zenith`.
+* Added `min_cos_zenith` and `max_zenith` keyword arguments to `erbs`.
+  (:issue:`681`)
```

Comment:
might need double ticks here (and below) for ReStructured Text - this always get's me - single ticks is for `markdown`
```rst
* func:`~pvlib.irradiance.erbs.erbs` ``doy`` argument changed to ``datetime_or_doy`` to be consistent with
  allowed types and similar functions (``disc``, ``get_extra_radiation``).
* func:`~pvlib.irradiance.erbs.erbs` DataFrame vs. OrderedDict return behavior now determined by type
  of ``datetime_or_doy`` instead of ``ghi`` or ``zenith``.
* Added ``min_cos_zenith`` and ``max_zenith`` keyword arguments to func:`~pvlib.irradiance.erbs.erbs`.
  (:issue:`681`)
```

Reply:
The Python docs seem to consistently use italics for parameters and links for functions. I don't think we have a standard here (though I recall you've done some work on this in older whats new files).

Reply:
I think in rst single ticks get interpreted as italics just like * do, anyway, not a blocker for me, so I defer to you, everything else looks great! Thanks for checking closure on `ghi = dni*cos(ze)+dhi` 😸